src/components/common/Button.tsx

```diff
@@ -45,7 +45,12 @@ const Button = forwardRef(
       )
 
     return (
-      <ChakraButton ref={ref} isLoading={isLoading} {...rest}>
+      <ChakraButton
+        ref={ref}
+        isLoading={isLoading}
+        loadingText={loadingText}
```

Comment:
Is this actually needed? We should display `loadingTest` in line 63 already 👀

Reply:
Jeje, without it, it doesn't show any loading text, just the spinner. It makes sense looking at the Chakra example: https://chakra-ui.com/docs/components/button/usage#button-loading-state
Maybe this behavior changed with a Chakra version? 🤔

Reply:
hmm, yeah, interesting, but it's fine then, thanks! 💯